src/depends.py

```diff
@@ -1,5 +1,6 @@
 #! python
 
+import subprocess
```

Comment:
can we place all the imports in the alphabetic order.

Reply:
There's an automated tool to do this. Please add an issue to research this tool and include it in the code_quality fab task.